apps/nextjs-website/src/components/Icon.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+  Facebook,
+  HelpOutlineOutlined,
+  Instagram,
+  LinkedIn,
+  Twitter,
+} from '@mui/icons-material';
+import Medium from './icons/Medium';
```

Comment:
Check out if it is the same icon:

```suggestion
import { MediumIcon } from '@pagopa/mui-italia/dist/icons/MediumIcon';
```